src/static_canvas.class.ts

```diff
@@ -1688,25 +1708,6 @@
     }
   });
 
-  /**
-   * Returns Object representation of canvas
-   * this alias is provided because if you call JSON.stringify on an instance,
-   * the toJSON object will be invoked if it exists.
-   * Having a toJSON method means you can do JSON.stringify(myCanvas)
-   * @function
-   * @param {Array} [propertiesToInclude] Any properties that you might want to additionally include in the output
-   * @return {Object} JSON compatible object
-   * @tutorial {@link http://fabricjs.com/fabric-intro-part-3#serialization}
-   * @see {@link http://jsfiddle.net/fabricjs/pec86/|jsFiddle demo}
-   * @example <caption>JSON without additional properties</caption>
-   * var json = canvas.toJSON();
-   * @example <caption>JSON with additional properties included</caption>
-   * var json = canvas.toJSON(['lockMovementX', 'lockMovementY', 'lockRotation', 'lockScalingX', 'lockScalingY']);
-   * @example <caption>JSON without default values</caption>
-   * canvas.includeDefaultValues = false;
-   * var json = canvas.toJSON();
-   */
-  fabric.StaticCanvas.prototype.toJSON = fabric.StaticCanvas.prototype.toObject;
```

Comment:
this was probably my fault.

Reply:
<img width="233" alt="image" src="https://user-images.githubusercontent.com/1194048/182468802-11d5a476-5df7-4fd5-be65-9b384f83a0af.png">

well not this time, but this looks totally like something i would do.

Reply:
11 years this bug and no one was armed

Reply:
then comes @ShaMan123 ...

Reply:
🤪